frontend/src/routes/Home/Search/searchDefinitions.tsx

```diff
@@ -1497,12 +1506,7 @@ export function CreateApplicationTopologyLink(item: any) {
         const link = `${NavigationPath.applicationTopology
             .replace(':namespace', item.namespace)
             .replace(':name', item.name)}?apiVersion=${apiversion}`
-        return (
-            <a href={link}>
-                {/* TODO Not translating - caused issue: https://github.com/open-cluster-management/backlog/issues/9184 */}
-                {'View topology'}
-            </a>
-        )
+        return <a href={link}>{'View topology'}</a>
```

Comment:
Have you investigated https://github.com/open-cluster-management/backlog/issues/9184 (mentioned in the comment you removed) and whether translating this string would cause an issue?